tasmota/tasmota_xsns_sensor/xsns_05_ds18x20.ino

```diff
@@ -62,7 +63,7 @@ struct {
   uint8_t valid;
   int8_t pins_id;
 #ifdef DS18x20_USE_ID_ALIAS
-  uint8_t alias;
+  char alias[17] = "0";
```

Comment:
That's a costly business. Time you make this struct virtual to save lots of memory. Rewrite to use calloc()

Reply:
Done